tests/instruments/keysight/test_keysight35670A_with_device.py

```
@@ -0,0 +1,295 @@
+        if token.strip()
+    }
+    return "AY6" in tokens or "A6J" in tokens
+
+
+@pytest.fixture(scope="module")
+def analyzer(connected_device_address):
+    """Create one instrument instance per module with safe teardown and queue drain."""
+    instr = Keysight35670A(connected_device_address, timeout=20000)
+    try:
+        instr.source_output_enabled = False
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)